invenio_records_ui/views.py

```diff
@@ -178,7 +178,7 @@ def record_view(pid_value=None, resolver=None, template=None,
     return view_method(pid, record, template=template, **kwargs)
 
 
-def default_view_method(pid, record, template=None):
+def default_view_method(pid, record, template=None, **kwargs):
```

Comment:
Probably fine to just access them from ``request.view_args`` - I'll just need to update previewer and files-rest. Will update the PR.